webapp/store/survey/actions/active.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+import { surveyCreate, surveyUpdate } from './actionTypes'
+
+export const setActiveSurvey = (surveyId, canEdit = true, dispatchSurveyCreate = false) => async (dispatch) => {
+  // Load survey
+  const params = { draft: canEdit, validate: canEdit }
+  const {
+    data: { survey },
+  } = await axios.get(`/api/survey/${surveyId}`, { params })
```

Comment:
Maybe it is not the right moment but we have axios spread around the code how about make (a function/set of functions) as a proxy of axios? I mean like `api.get` and this Api object wrapper a call to axios, in that way it is easier to replace axios.

Reply:
yes but not now